airflow/providers/amazon/aws/hooks/logs.py

```diff
@@ -60,6 +61,7 @@ def get_log_events(
         log_group: str,
         log_stream_name: str,
         start_time: int = 0,
+        end_time: int | None = None,
```

Comment:
should there be also an updated test case in `TestAwsLogsHook`?

Reply:
I can add one but to me, it is not necessary, we are just passing the value down to `get_cloudwatch_logs`. But since it had been called out by @o-nikolas as well, I'll add one

Reply:
Done